backend/apps/slack/common/handlers/ai.py

```
@@ -30,12 +30,108 @@ def get_blocks(
+        text (str): The AI response text.
+
+    Returns:
+        list: A list of Slack blocks.
+
+    """
+    # Format the AI response for Slack (remove code blocks, fix markdown)
+    formatted_response = format_ai_response_for_slack(text)
+    
+    # Slack has a limit of 3001 characters per block text
+    # Split into multiple blocks if needed
+    MAX_BLOCK_TEXT_LENGTH = 3000  # Leave some margin for safety
```

Ruff (0.14.14)

[warning] Variable `MAX_BLOCK_TEXT_LENGTH` in function should be lowercase

(N806)